### DISCUSSION TITLE
CCO Dashboard GOV - Error 0x80040E4E

### DISCUSSION THREAD

**Discussion by wasytb72** [Azure/CCOInsights#181] [Q&A] [answered]
All the pre-req are met but the dashboard is not loading. OLEDC or ODBC exception 0x80040E4E in many of the tables.

Besides that, CC Entities failed with error [Expression.Error] cannot convert value Error retrieving data.

Same device / same PowerPI version / settings against other tenant is working just fine.

Anybody has experienced this scenario?

Thanks

**Comment by piquibr** [comment]
Same error here. Could you find the way to fix this?

**Comment by simonesavi** [comment]
![image](https://user-images.githubusercontent.com/16479009/170514040-b93e4c70-2e1c-487e-9ca8-cb4404c76a93.png)

same error

**Comment by simonesavi** [comment, marked as answer]
After some tests, this is a potential solution:

- Close the error window
- Click on "Discard changes"
![image](https://user-images.githubusercontent.com/16479009/170677110-95d019d0-9a26-4aec-8599-06861faf76dd.png)
- On the window that will appear, click on "Discard"
![image](https://user-images.githubusercontent.com/16479009/170680370-987a5bd3-d9b8-49cc-9bd7-11945a63314a.png)


Refresh data and the problem should disappear
